packages/shared/sdk-client/src/api/LDOptions.ts

```diff
@@ -139,6 +139,14 @@ export interface LDOptions {
    */
   logger?: LDLogger;
 
+  /**
+   * The maximum number of locally cached contexts.  Local caches hold data for use in situations
+   * where the SDK cannot reach the LaunchDarkly services.
```

Comment:
Maybe something like:
```suggestion
   * The maximum number of locally cached contexts.  The cache is used to decrease initialization
   * latency and to provide fallback when the SDK cannot reach LaunchDarkly services.
```
But I need to review the rest of the PR before I know if that is true.